src/__experimental__/components/date/date.spec.js

```diff
@@ -178,11 +177,27 @@ describe('Date', () => {
       });
 
       describe('and with DatePicker opened', () => {
-        it('then onBlur prop should have been called', () => {
+        it('then onBlur prop should not have been called', () => {
           simulateFocusOnInput(wrapper);
           wrapper.setProps({ value: secondDate });
           expect(wrapper.find(DatePicker).exists()).toBe(true);
-          expect(onBlurFn).toHaveBeenCalled();
+          expect(onBlurFn).not.toHaveBeenCalled();
+        });
+      });
+
+      describe('and the rawValue is invalid', () => {
+        it('then onBlur prop should not have been called', () => {
```

Comment:
it block text does not match the content.

Reply:
Yep good spot, will fix